tests/test_mailgun_backend.py

```diff
@@ -448,7 +448,11 @@ def test_tracking(self):
         self.assertEqual(data['o:tracking-opens'], 'no')
         self.assertEqual(data['o:tracking-clicks'], 'yes')
 
-    # template_id: Mailgun doesn't support stored templates
+    def test_template_id(self):
+        self.message.template_id = "welcome_template"
+        self.message.send()
+        data = self.get_api_call_json()
```

Comment:
Changing this to `self.get_api_call_data()` should make the tests pass. (Mailgun's API uses form field encoding, rather than JSON.)

Reply:
Whoops, thanks! Updated.